src/Blocks/components/rating/rating.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/**
+ * Template for the Rating Component.
+ *
+ * @package EightshiftForms
+ */
+
+use EightshiftForms\Helpers\Helper;
+use EightshiftFormsVendor\EightshiftLibs\Helpers\Components;
+
+$manifest = Components::getManifest(__DIR__);
+$icons = Components::getComponent('utils')['icons'];
+$manifestGlobal = Components::getSettings();
+$manifestCustomFormAttrs = $manifestGlobal['customFormAttrs'];
+$manifestTypeInternal = $manifestGlobal['typeInternal'];
+
+$componentClass = $manifest['componentClass'] ?? '';
+$componentJsClass = $manifest['componentJsClass'] ?? '';
+$additionalClass = $attributes['additionalClass'] ?? '';
+
+$ratingName = Components::checkAttr('ratingName', $attributes, $manifest);
+if (!$ratingName) {
+	return;
+}
+
+$ratingValue = Components::checkAttr('ratingValue', $attributes, $manifest);
+$ratingTypeCustom = Components::checkAttr('ratingTypeCustom', $attributes, $manifest);
+$ratingIsDisabled = Components::checkAttr('ratingIsDisabled', $attributes, $manifest);
+$ratingIsReadOnly = Components::checkAttr('ratingIsReadOnly', $attributes, $manifest);
+$ratingIsRequired = Components::checkAttr('ratingIsRequired', $attributes, $manifest);
+$ratingTracking = Components::checkAttr('ratingTracking', $attributes, $manifest);
+$ratingAttrs = Components::checkAttr('ratingAttrs', $attributes, $manifest);
+$ratingFieldAttrs = Components::checkAttr('ratingFieldAttrs', $attributes, $manifest);
+$ratingAmount = Components::checkAttr('ratingAmount', $attributes, $manifest);
+$ratingHideLabel = false;
+
+$ratingClass = Components::classnames([
+	Components::selector($componentClass, $componentClass),
+	Components::selector($componentJsClass, $componentJsClass),
+	Components::selector($additionalClass, $additionalClass),
+]);
```

Comment:
```suggestion
$ratingClass = Components::classnames([
	$componentClass,
	$componentJsClass,
	$additionalClass,
]);
```

Reply:
Tnx for the suggestion but I will leave it as it